Source/Core/DolphinQt/Settings/BroadbandAdapterSettingsDialog.cpp

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "DolphinQt/Settings/BroadbandAdapterSettingsDialog.h"
+
+#include <regex>
+#include <string>
+
+#include <QDialog>
```

Comment:
Since this is included in the header we don't need it here.

Other than that, LGTM.

Reply:
Done